Document Slack agent cross-repo access (#5541)
mintlify/docs: agent/slack.mdx

The agent reads the context of the request or thread and creates a pull request in your connected repository with the updates.

## Reference other repositories

> The agent can read source code from any repository where the Mintlify GitHub App is installed, even if it isn't your documentation repository. Use this to ground documentation updates in code that lives in a separate repo.

Reviewer: ```suggestion
The agent can read source code from any repository that has the Mintlify GitHub App installed, even if it isn't your documentation repository. This lets the agent build deeper context to guide documentation updates.
```

When you mention another repository in your request, the agent clones it on demand and inspects the relevant files before drafting changes.

Example prompts:

- "Update the authentication guide based on the changes in `acme/api-server/src/auth.ts`."
- "Document the new endpoints added in `acme/backend#1234`."
- "What repositories do you have access to?"

> Cloned repositories are read-only. The agent uses them for context, but pull requests are always opened against your documentation repository.

Reviewer: ```suggestion
Cloned repositories are read-only. The agent uses them for context, but always opens pull requests against your documentation repository.
```

> If the agent reports that the GitHub App is not installed on the referenced owner, install the [Mintlify GitHub App](https://github.com/apps/mintlify/installations/new) on that organization or user account and try again.

Reviewer: ```suggestion
```
